analyzer/codechecker_analyzer/analyzers/clangsa/config_handler.py

```diff
@@ -32,6 +33,26 @@ def __init__(self, environ):
         self.environ = environ
         self.version_info = None
 
+    @property
+    def analyzer_plugins(self):
+        """ Full path of the analyzer plugins. """
```

Comment:
Will this return an empty list if `CC_ANALYZERS_FROM_PATH` is set to true but the `CC_CLANGSA_PLUGIN_DIR` is not set?
I think in this case it will return the plugin from the package.
The plugin_dir exists and it is set so it will not return here.
```py
        if not plugin_dir or not os.path.exists(plugin_dir):
            return []
```

Reply:
This is not possible because if the `CC_ANALYZER_FROM_PATH` is set the `plugin_dir` will be set to `None`:
https://github.com/Ericsson/codechecker/blob/2d9ffc3ff6d187d9c65e4d25b6b5d63fda95767e/analyzer/codechecker_analyzer/analyzer_context.py#L248-L251
However you are right, so I updated this function to handle this use case too.

Reply:
Thanks for the extension!